shell/app/modules/workBench/pages/projects/project-list-protocol.tsx

```diff
@@ -75,6 +76,10 @@ export const ProjectList = () => {
   );
 };
 
+export const ProjectListMy = () => <ProjectList scenarioKey='project-list-my'
+  scenarioType='project-list-my' key='project-list-my' />
+export const ProjectListAll = () => <ProjectList scenarioKey='project-list-all'
+  scenarioType='project-list-all' key='project-list-all' />
```

Comment:
How about to check route path equals `projects ` or `public-projects` to distinguish these two components, in the future, it's better to export route level Comp by default export